model/client4.go

```diff
@@ -479,6 +483,17 @@ func (c *Client4) GetAudits(userId string, page int, perPage int, etag string) (
 	}
 }
 
+// AutocompleteUsers returns a page of users on a team based on search term
+func (c *Client4) AutocompleteUsers(teamId string, channelId string, username string, etag string) ([]*User, *Response) {
```

Comment:
Once you add the cases above you'll need a different driver function for the three cases